typegraph/tests/test_sdks.py

```diff
@@ -0,0 +1,51 @@
+# Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+# SPDX-License-Identifier: MPL-2.0
+
+import subprocess
+from pathlib import Path
+import sys
+
+
+def serialize(path: Path) -> str:
+    print(f"serializing {path}...")
+    proc = subprocess.run(
+        [
+            "cargo",
+            "run",
+            "-p",
+            "meta-cli",
```

Comment:
would be great if the SDK tests are controlled/written in Rust, so no multiple test framework to maintain or run (but if this adds too much overhead because of the types), we may fallback on all deno tests instead.